source/_integrations/keyboard_remote.markdown

```diff
@@ -165,3 +165,32 @@ You can check ACLs permissions with:
 ```bash
 getfacl /dev/input/event*
 ```
+
+## Containers
+
+If you are running Home Assistant inside of a Docker container, you need to pass the input device through to the container. This can be done by running the container in privileged mode, but this is is not ideal for security. Instead you can pass the input device you want to use directly into the container with the `--devices` flag. However, restarting the container or unplugging and replugging your keyboard will break this integration as only the instance of the keyboard that existed when the container first starts will be avaible inside of the container.
```

Comment:
Please reference this as the "Home Assistant Container" installation method. Docker is just one of the many available.